generator/tree.go

```diff
@@ -115,6 +115,13 @@ func prepareTree(nodes *Node) map[string]*Node {
 		}
 	})
 
+	// Convert RFC 2579 DateAndTime textual conversion to type.
+	walkNode(nodes, func(n *Node) {
```

Comment:
This and the previous two walks should really be combined

Reply:
I saw this, and was wondering why we did walkNode over more than once. I can refactor.